packages/api-explorer/jest-puppeteer.config.js

```diff
@@ -36,9 +37,9 @@ module.exports = {
     },
   },
   browserContext: 'default',
-  server: {
-    command: 'yarn develop',
-    port: 8080,
-    launchTimeout: 50000,
-  },
+  // server: {
```

Comment:
Delete commented out code?

Reply:
Thanks for catching. I needed to revert my debugging changes. @josephaxisa and I discussed having a "debug mode" for puppeteer also